CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Fixed
```

Comment:
This technically is a `### Breaking Change` as it crosses major version boundaries and could cause other gems that are still reliant on `< 2.0`. There are some pretty significant changes regarding filters.